src/WebJobs.Extensions.DurableTask/Correlation/NoOpTelemetryChannel.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using Microsoft.ApplicationInsights.Channel;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Correlation
+{
+    /// <summary>
+    /// A stub of <see cref="ITelemetryChannel"/>.
+    /// </summary>
+    public sealed class NoOpTelemetryChannel : ITelemetryChannel
+    {
```

Comment:
I'm not entirely sure why we need this class. Once we find out, we should add it to the class header